Content.Shared/GameObjects/EntitySystems/ActionBlockerSystem.cs

```diff
@@ -161,5 +171,15 @@ public static bool CanChangeDirection(IEntity entity)
 
             return canchangedirection;
         }
+
+        public static bool CanShiver(IEntity entity)
```

Comment:
Given that CanShiver exists now, you might as well add CanSweat as well, to avoid dead bodies from cooling themselves down through sweating for instance.

Reply:
I added a condition to disable metabolism at all if state == dead